packages/@sanity/base/src/preview/observeWithPaths.js

```diff
@@ -18,17 +19,51 @@ function listen(id) {
     .filter(event => event.documentId === id)
 }
 
-function fetchAllDocumentSnapshots(selections) {
-  const optimizedParams = {}
-  const queryParts = selections.map(([id, paths], queryIndex) => {
-    optimizedParams[[`id_${queryIndex}`]] = id
-    return `*[_id==$id_${queryIndex}]{_id,_type,${paths.join(',')}}`
-  })
+function combineSelections(selections) {
+  return Object.values(selections.reduce((output, [id, paths], index) => {
```

Comment:
`Object.values()` is not supported in IE11, perhaps use `values` from lodash?

Reply:
Good idea. Will fix!